subprojects/docs/src/docs/userguide/userguide_single.adoc

```diff
@@ -23,195 +23,243 @@
+include::build_environment.adoc[leveloffset=+2]
+include::gradle_daemon.adoc[leveloffset=+2]
+include::file_system_watching.adoc[leveloffset=+2]
+include::init_scripts.adoc[leveloffset=+2]
 
-include::more_about_tasks.adoc[leveloffset=+2]
+[[part:authoring_builds]]
+== **AUTHORING GRADLE BUILDS**
 
-include::incremental_build.adoc[leveloffset=+2]
+== LEARNING THE BASICS
 
+include::build_lifecycle.adoc[leveloffset=+2]
+include::directory_layout.adoc[leveloffset=+2]
+include::tutorial_using_tasks.adoc[leveloffset=+2]
 include::writing_build_scripts.adoc[leveloffset=+2]
-
-include::working_with_files.adoc[leveloffset=+2]
-
 include::plugins.adoc[leveloffset=+2]
-
-include::build_lifecycle.adoc[leveloffset=+2]
-
+include::working_with_files.adoc[leveloffset=+2]
 include::logging.adoc[leveloffset=+2]
+include::potential_traps.adoc[leveloffset=+2]
 
-include::multi_project_builds.adoc[leveloffset=+2]
+== STRUCTURING INDIVIDUAL BUILDS
 
+include::multi_project_builds.adoc[leveloffset=+2]
+include::declaring_dependencies_between_subprojects.adoc[leveloffset=+2]
 include::sharing_build_logic_between_subprojects.adoc[leveloffset=+2]
-
 include::fine_tuning_project_layout.adoc[leveloffset=+2]
+include::multi_project_configuration_and_execution.adoc[leveloffset=+2]
 
-include::declaring_dependencies_between_subprojects.adoc[leveloffset=+2]
+== STRUCTURING SOFTWARE PRODUCTS
 
-include::multi_project_configuration_and_execution.adoc[leveloffset=+2]
+include::structuring_software_products.adoc[leveloffset=+2]
+include::structuring_software_products_details.adoc[leveloffset=+2]
+include::composite_builds.adoc[leveloffset=+2]
 
-include::organizing_gradle_projects.adoc[leveloffset=+2]
+== AUTHORING SUSTAINABLE BUILDS
 
+include::organizing_gradle_projects.adoc[leveloffset=+2]
 include::authoring_maintainable_build_scripts.adoc[leveloffset=+2]
 
+== DEVELOPING GRADLE TASKS
+
+include::more_about_tasks.adoc[leveloffset=+2]
+include::incremental_build.adoc[leveloffset=+2]
+include::custom_tasks.adoc[leveloffset=+2]
 include::lazy_configuration.adoc[leveloffset=+2]
+include::worker_api.adoc[leveloffset=+2]
 
-include::test_kit.adoc[leveloffset=+2]
+== DEVELOPING GRADLE PLUGINS
 
-include::ant.adoc[leveloffset=+2]
+include::custom_plugins.adoc[leveloffset=+2]
+include::designing_gradle_plugins.adoc[leveloffset=+2]
+include::implementing_gradle_plugins.adoc[leveloffset=+2]
+include::testing_gradle_plugins.adoc[leveloffset=+2]
+include::publishing_gradle_plugins.adoc[leveloffset=+2]
 
-[[part:dependency_management]]
-== Dependency Management
+== OTHER DEVELOPING GRADLE TOPICS
 
-include::dependency_management.adoc[leveloffset=+2]
+include::custom_gradle_types.adoc[leveloffset=+2]
+include::build_services.adoc[leveloffset=+2]
+include::dataflow_actions.adoc[leveloffset=+2]
+include::test_kit.adoc[leveloffset=+2]
+include::ant.adoc[leveloffset=+2]
 
 [[part:jvm_projects]]
-== Java & Other JVM Projects
+== **AUTHORING JVM BUILDS**
```

Comment:
💅 These bolded sections don't really look very different in the rendered HTML.  Could something else be done to make them stand out?

Reply:
Agreed. I looked into this and the asciidoctor docs and could not find a solution. I can't use custom HTML because of the PDF rendering and the toc creation is very restrictive.